### ISSUE TITLE
Only one partition can use index when we create index by partition.

### ISSUE THREAD

**Issue by LuciferYang** [Intel-bigdata/OAP#1038]
In production environment we do `create oindex userid_index on table(userid) parttiion(p1=xxx,p2=yyy,p3=zzz)` and index data layout as following:

```
/day=20190411/event=16/min=05/part0-userid-${time1}.index

/day=20190411/event=16/min=05/.oap.meta( record ${time1})

/day=20190411/event=16/min=10/part0-userid-${time2}.index

/day=20190411/event=16/min=10/.oap.meta(record${time2})
```

then if we query as `select xxx from table  userid = xxx` there is only one partition datas can actual use userid index at Executor side because we push down IndexScaner(userid) with time1 or time2.

**Comment by LuciferYang** [comment]
The issue that caused this problem is #267,  @adrian-wang fix a bug describe in #267, and after #480 risks have been reduced, but they may still exist in some scenes, discuss with @LinhongLiu offline, we choice a simply fix method: use a constant timestamps to workaround and make sure the following scenarios do not occur in production environment:
```
create oindex index1 on table(a)

query with column a and index will be cache by executor side

drop oindex index1 on table

create oindex index1 on table(b)
```

**Comment by LuciferYang** [comment]
#1039 fix this
